source/_integrations/airtouch5.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: AirTouch 5
+description: Instructions on how to integrate the AirTouch 5 A/C controller into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2023.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@danzel'
+ha_domain: airtouch5
+ha_platforms:
+  - binary_sensor
+  - climate
+  - cover
+ha_integration_type: integration
+---
+
+The AirTouch 5 integration allows you to control Ducted Air Conditioning Systems that are using the [AirTouch 5](https://www.airtouch.net.au/smart-air-conditioning/airtouch-5/) Controller.
+
+{% include integrations/config_flow.md %}
+
+## Entities
+
+### Climate
+
+The integration will create a climate entity for each AC unit and each zone.
+
+### Cover
+
+Each zone has a cover entity to represent the state of the damper. The damper position can be controlled only if the zone is not temperature controlled.
+
+### Binary Sensor
+
+Each AC unit has binary sensors to show whether bypass or spill is active.
+
+Each zone has binary sensors to show if the battery is low or spill is active.
```

Comment:
```suggestion
```

They will not be added with the linked PR. Please create a separate docs PR for them